### PR TITLE
include OK Panel

### PR DESCRIPTION
**Pull request by smth**

### CODE REVIEW

File: a/html/oki/panel/index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            </div>
+          </div>
+        </aside>
+      </div>
+    </div>
+
+    <footer class="site-footer">
+      <div class="container">
+        <a class="footer-logo" href="https://okfn.org/">
+          <img src="https://bytebucket.org/okfn/assets.okfn.org/raw/88b24904b8ecded5e6d530739743162d1c5b3d93/p/okfn/img/okfn-logo-landscape-white.png" alt="Open Knowledge">
```
Comment (pwalsh) [inline a/html/oki/panel/index.html]: @smth ?

Comment (smth) [inline a/html/oki/panel/index.html]: Hello?

Comment (pwalsh) [inline a/html/oki/panel/index.html]: Link to an old asset no?

Comment (smth) [inline a/html/oki/panel/index.html]: Ah yes, sorry (I was confused by the Github interface for a moment there). I'll correct that.

**Pull request merged**